schedule-works-be/dgraph.py

```diff
@@ -1,67 +1,32 @@
-"""Functions for interacting with directed graph structure."""
-
 import networkx as nx
 import matplotlib.pyplot as plt
 
-
-def course_sum(graph: nx.DiGraph, course: str, memo: dict) -> set:
+def topological_sort(graph: nx.DiGraph) -> list:
     """
-    Determine a given course's dependency list. See make_priority_queue() to use properly.
-
-    Wrapper function to clarify args used during recursion.
-    Functions best when courses are fed in order from least expected dependency to most.
-
+    Return a list of nodes in topological order.
 
     Args:
-    G: A directed graph containing courses and edges that point towards a courses prequisite.
```

Comment:
Why'd you remove all these functions here

Reply:
I had errors when I kept it, and my topo function sort of did it's role for them. I think I should've kept it tbh